### DISCUSSION TITLE
Python license scanning

### DISCUSSION THREAD

**Discussion by kanton10062006** [aquasecurity/trivy#6314] [Q&A] [answered]
### Question

Hello team,

Are you going to implement license scanning for Python package managers in the near future?
As I can see for now license detection is not supported for package managers.

Thanks and looking forward to your reply.

### Target

None

### Scanner

License

### Output Format

None

### Mode

None

**Comment by kanton10062006** [comment]
Anyone?

**Comment by DmitriyLewen** [comment, marked as answer]
Hello @kanton10062006 
Sorry for waiting.

You are right. We don't support license scanning for Python package managers.

We don't plan to add support for this anytime soon. 
But if you have any ideas for finding licenses (if I remember correctly, pip/Pipenv/Poetry lockfiles don't contain license information) - tell us. This will speed up adding this.
We also always welcome new contributors - if you have time to create PR - we will check your changes and merge them.

Regards, Dmitriy

**Reply by kanton10062006** [reply]
Hey Dmitry,

Gotcha. Thanks for the update

Correct, pip/Pipenv/Poetry lockfiles don't contain license information and it could be quite tricky to get such information.
We accomplished it using a simple Python package that executes pip show against dependencies and parses the license from pip's output.
I wish but it seems like there is no easy and better way to implement it.

Thanks
